### DISCUSSION TITLE
[FEATURE REQUEST] Additional monetary donation services outside of Liberapat

### DISCUSSION THREAD

**Discussion by kkm000** [Cookie-AutoDelete/Cookie-AutoDelete#1377] [Ideas / Feature Requests] [answered]
### Acknowledgements

- [X] I acknowledge that I have read the above items

### Is your feature request related to a problem? Please describe.

Liberapay is a little-known service. They support neither SSO login with any OAuth provider (Google, Twitter, GitHub(!)), nor 2FA. I tried to donate using it, but dropped my attempt halfway through, there were web site navigation hiccups, and I already had my mindset against, and decided not to attempt to use them any further. I do not remember the actual issue I encountered, but, simply speaking, I did not trust an obscure, isolated service without any reputation and located overseas from the US.

### Describe the solution that you'd like

* The best known are likely Patreon and Paypal.
* Having both one-time and recurring options is likely a boon.

**Comment by kennethtran93** [comment, marked as answer]
There was initially PayPal, but it was tied to the one creator/dev at that time.  When I joined on, we needed a different platform where the fees were minimal and it would automatically split the payments out between us (well my portion from liberapay is transferred to PayPal anyway).  Yes we realize that their site is not in the US.  Then again, I'm not from the US either.

While liberapay doesn't state 'one-time', you can simply do a 'manual recurrence' and just pay one-time.

As far as Patreon is concerned, they most likely have fees as well.

**Reply by farblos** [reply]
I second @kkm000, using liberapay feels slightly awkward, to say the least. Anyway, the paypal backend has deducted 10EUR in your favor, so I hope this reaches you as intended. BTW, my favorite feature request would be this sync thingy ... and thanks for maintaining CAD!

**Comment by kennethtran93** [comment]
Migrating this to discussion as I believe it fits better as a discussion rather than an issue.
